### PR TITLE
Launch session dropdown

### PR DESCRIPTION
**Pull request by VoxSecundus**
NB: This PR is based off of #83 and includes changes made on that branch. It should be rebased once #83 has been merged into `master.`

This PR includes work done to convert a pair of buttons on the `BrandBar` used to launch new desktop sessions into a single, split dropdown button. The main button has the old `Launch new session` functionality (takes you to `/sessions/new`); the one dropdown item is `Quick launch session`, which retains the same functionality (tries to launch a one-click session of whatever Flight Desktop's default is).

### CODE REVIEW

File: src/LaunchDropdown.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import { ButtonDropdown, DropdownToggle, DropdownMenu } from 'reactstrap';
+import QuickLaunchButton from './QuickLaunchButton';
+
+function LaunchDropdown({ className }) {
+  const [dropdownOpen, setOpen] = React.useState(false);
+  const toggle = () => setOpen(!dropdownOpen);
+
+  return <ButtonDropdown isOpen={dropdownOpen} toggle={toggle}>
+    <Link
+      className={className}
```
Comment (benarmston) [inline src/LaunchDropdown.js:13]: Can we add `dropdown` to the classes here.  It fixes a styling issue.  

image

vs

image

**Review by benarmston — CHANGES_REQUESTED** [review]
Can you look at the below @VoxSecundus and whilst doing that also change "My sessions" to "Sessions".

File: src/LaunchDropdown.js (hunk 1 above)
Comment (VoxSecundus) [reply src/LaunchDropdown.js:13]: Just adding `dropdown` doesn't change anything for me

**Review by benarmston — COMMENTED** [review]
FSR, quick launching the desktop session now gives a spurious error message about the desktop not having been configured.  "Slow launching" does not.

Investigation needed.

**Comment by VoxSecundus** [comment]
> FSR, quick launching the desktop session now gives a spurious error message about the desktop not having been configured. "Slow launching" does not.

The same occurs on the `master` branches of `flight-desktop-webapp`/`flight-landing-page`/`flight-webapp-components`.

**Pull request merged**